config/runtime.exs

```diff
@@ -191,12 +216,39 @@ if config_env() != :test do
     metrics_jwt_secret: System.get_env("METRICS_JWT_SECRET"),
     db_enc_key: System.get_env("DB_ENC_KEY"),
     region: System.get_env("REGION"),
-    prom_poll_rate: System.get_env("PROM_POLL_RATE", "5000") |> String.to_integer(),
-    platform: platform,
+    prom_poll_rate: Env.get_integer("PROM_POLL_RATE", 5000),
     slot_name_suffix: slot_name_suffix
+end
+
+# Setup Production
 
-  queue_target = System.get_env("DB_QUEUE_TARGET", "5000") |> String.to_integer()
-  queue_interval = System.get_env("DB_QUEUE_INTERVAL", "5000") |> String.to_integer()
+if config_env() == :prod do
+  config :libcluster, debug: false, topologies: cluster_topologies
+  secret_key_base = System.fetch_env!("SECRET_KEY_BASE")
+  if app_name == "", do: raise("APP_NAME not available")
+
+  config :realtime, RealtimeWeb.Endpoint,
+    server: true,
+    url: [host: "#{app_name}.supabase.co", port: 443],
+    http: [
+      compress: true,
+      port: Env.get_integer("PORT", 443),
```

Comment:
should these ports change?

Reply:
I would say so. any reason why not do it for prod?

Reply:
🤦